solidity/contracts/token/HypERC20Collateral.sol

```diff
@@ -63,6 +65,20 @@ contract HypERC20Collateral is FungibleTokenRouter, MovableCollateralRouter {
         return wrappedToken.balanceOf(_account);
     }
 
+    function quoteTransferRemote(
+        uint32 _destinationDomain,
+        bytes32 _recipient,
+        uint256 _amount
+    ) external view virtual override returns (Quote[] memory quotes) {
+        quotes = new Quote[](2);
+        quotes[0] = _quoteTransferRemote(
+            _destinationDomain,
+            _recipient,
+            _amount
+        )[0];
+        quotes[1] = Quote({token: address(wrappedToken), amount: _amount});
```

Comment:
This seems to imply that the second fee is the whole amount amount of the token being transferred.

Reply:
I think we should document this behavior because this probably wasn't most folks' interpretation of the original design doc. Probably would specify this at least on this implementation as well as the `ITokenBridge` interface